ios/ReactNativeAdsFacebook.xcodeproj/project.pbxproj

```diff
@@ -305,7 +305,7 @@
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
 					"$(PROJECT_DIR)/**",
-					"$(SRCROOT)/../../../ios/Pods/FBAudienceNetwork",
+					"$(SRCROOT)/../../../ios/Pods/FBAudienceNetwork/**",
```

Comment:
The up-to-date Pod does not exist in `$(SRCROOT)/../../../ios/Pods/FBAudienceNetwork`. It exists in `$(SRCROOT)/../../../ios/Pods/FBAudienceNetwork/Static`. This change will not break the previous versoins.